addons/website/views/website_templates.xml

```diff
@@ -270,6 +274,7 @@
                     </div>
                 </div>
             </section>
+            <t t-raw="website.custom_code_footer"/>
```

Comment:
The content of an oe_structure is fully editable so here you will prevent correct edition of the footer by adding this. I think you should add this code at the end of the body, after the footer.